lib/class-wp-theme-json-gutenberg.php

```diff
@@ -2633,6 +2641,26 @@ static function ( $pseudo_selector ) use ( $selector ) {
 				}
 				unset( $declarations[ $index ] );
 			}
+
+			if ( $is_root_selector && ( 'background-image' === $declaration['name'] || 'background' === $declaration['name'] ) ) {
+				$should_set_root_min_height = true;
+			}
+		}
+
+		/*
+		 * If root styles has backgroundSize === cover, or a background is set (gradient),
+		 * set the body minHeight to 100%.
```

Comment:
Nit: use CSS property names since we're in PHP:

```suggestion
		 * If root styles has a background-image, or a background is set (gradient),
		 * set the root/html min-height to 100%.
```

Would it also be worth adding some of the extra info you've added in the comment about why to use `html` or `body`, or would that make the comment too verbose? Just thought it might help for our future selves if folks wonder why it's `html` instead of `body`.